src/main/java/org/broadinstitute/dsm/careevolve/Covid19OrderRegistrar.java

```diff
@@ -0,0 +1,302 @@
+
+                    JsonObject latestKitActivity = getLatestKitActivity(activities);
+                    JsonArray latestKitAnswers = latestKitActivity.get("questionsAnswers").getAsJsonArray();
+
+                    String collectionDateTimeStr = null;
+                    Instant collectionDateTime = null;
+                    StringBuilder collectionDate = new StringBuilder();
+                    StringBuilder collectionTime = new StringBuilder();
+                    for (int i = 0; i < latestKitAnswers.size(); i++) {
+                        JsonObject latestKitAnswer = latestKitAnswers.get(i).getAsJsonObject();
+                        String questionStableId = latestKitAnswer.get("stableId").getAsString();
+
+                        if ("SAMPLE_COLLECT_DATE".equals(questionStableId)) {
+                            collectionDate.append(latestKitAnswer.get("date").getAsString());
+                        } else if ("SAMPLE_COLLECT_TIME".equals(questionStableId)) {
+                            JsonArray timeFields = latestKitAnswer.getAsJsonArray("answer").get(0).getAsJsonArray();
+                            for (int timeFieldIdx = 0; timeFieldIdx < timeFields.size(); timeFieldIdx++) {
+                                collectionTime.append(" ").append(timeFields.get(timeFieldIdx).getAsString());
+                            }
+                        }
+                    }
+                    collectionDateTimeStr = collectionDate + " "  + collectionTime;
+                    try {
+                        collectionDateTime = new SimpleDateFormat("yyyy-MM-dd h m a").parse(
+                                collectionDateTimeStr
+                        ).toInstant();
+                    } catch (ParseException e) {
+                        throw new CareEvolveException("Could not parse collection date time " + collectionDateTimeStr.toString()
+                                + " for participant " + patientId + " in activity instance " + latestKitActivity.get("guid"), e);
+                    }
+
+                    JsonObject baselineCovidActivity = getBaselineCovidActivity(activities);
+                    JsonArray baselineCovidAnswers = baselineCovidActivity.get("questionsAnswers").getAsJsonArray();
+                    String dob = null;
+
+                    // mappings  for  race, ethnicity, and  sex are custom  for the  Ellkay API
+                    // and are controlled  by the Ellkay integration
+                    String race = "2131-1"; // code for "other"
+                    String ethnicity = "U";
+                    String sex = "U";
+                    for (int i = 0; i < baselineCovidAnswers.size(); i++) {
+                        JsonObject answer = baselineCovidAnswers.get(i).getAsJsonObject();
+                        String questionStableId = answer.get("stableId").getAsString();
+
+                        if ("DOB".equals(questionStableId)) {
```

Comment:
D'oh! I just realized we're not exporting the birth_date in profile to ES. I'm making a change in TestBoston to copy `DOB` answer to birth_date so we have that there. Might be easier to grab from that instead of from the activity answers itself.

Reply:
Just realized that we _are_ exporting birth date already under `dsm.dateOfBirth`. I'll recommend grabbing from that but the current way is fine too.